pkg/sentry/arch/arch_aarch64.go

```diff
@@ -34,6 +34,9 @@ const (
 	SyscallWidth = 4
 )
 
+// ARMTrapFlag is an exported const for use by other packages.
```

Comment:
Please explain what it actually is rather than just that it is exported.

Reply:
Done!